devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,23 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu: MediaTek MT7628AN
+cpu_cores: 1
+cpu_mhz: 580
+memory_mb: 64
```

Comment:
Several devices support multiple cases of installed memory by autodetection (e.g. ath79/ramips).

For those, we might have e.g. 64/128 MiB variants supported with one image, i.e. appearing as one device as seen by OpenWrt. We should find a way to express this; of course we might just put "64/128", but that might break machine-readability. Can we just use the same variable name twice, or is there something like an array (I'm yaml noob)?

I'm not sure whether the latter case really helps much with machine-readability, though.

Reply:
It's possible to dynamically switch between integers and arrays of integers. That means more code for validation and e.g. rendering, but is possible.

I'm against any kind of device merging if *they are somewhat similar called different and have these random differences*, in these cases there should just be two files.